schemas/com.roku/device_info/jsonschema/1-0-0

```diff
@@ -0,0 +1,159 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema with information about Roku device (reflects the ifDeviceInfo interface: https://developer.roku.com/en-gb/docs/references/brightscript/interfaces/ifdeviceinfo.md)",
+  "self": {
+    "format": "jsonschema",
+    "name": "device_info",
+    "vendor": "com.roku",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "model": {
+      "type": "string",
+      "description": "Model name of the Roku device.",
+      "maxLength": 255
+    },
+    "modelDisplayName": {
+      "type": "string",
+      "description": "Model display name.",
+      "maxLength": 65535
+    },
+    "modelType": {
+      "type": "string",
+      "description": "Type of device.",
+      "maxLength": 255
+    },
+    "osVersion": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "Major, minor, revision, and build number of the Roku OS running on the device separated by dots.",
+      "maxLength": 255
+    },
+    "channelClientId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "A unique device identifier that is different across channels.",
+      "maxLength": 255
+    },
+    "isRIDADisabled": {
+      "type": [
+        "boolean",
+        "null"
+      ],
+      "description": "Indicates whether tracking via Roku's ID for Advertisers (RIDA) is disabled on the device."
+    },
+    "RIDA": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "A persistent unique identifier (UUID) for the device.",
+      "maxLength": 255
+    },
+    "captionsMode": {
+      "type": "string",
+      "description": "Determines whether global captions are turned on or off, or are in instant replay mode.",
+      "maxLength": 255
+    },
+    "audioOutputChannel": {
+      "type": "string",
+      "description": "Type of audio output.",
+      "maxLength": 255
+    },
+    "memoryLevel": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "General memory level of the channel.",
+      "maxLength": 255
+    },
+    "timeSinceLastKeypress": {
+      "type": "integer",
+      "description": "The number of seconds since the last remote keypress was received.",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "userCountryCode": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "ISO 3166-1 (2-letter) country code associated with the user's Roku account.",
+      "maxLength": 255
```

Comment:
The problem is that although normally the string should be 2 characters long, there are cases when it can be longer – according to the Roku documentation, for some specific channels (apps) that are published in some weird way that I don't really understand, the string can also be for example "Sky Germany", or "Econet". They promise to only return the 2-char long code in the future, but currently it is not guaranteed to be 2-char long.

Reply:
Hey @matus-tomlein do you think the description should then be changed?  Currently it says `ISO 3166-1 (2-letter)`.  Something like `Country code associated with the user's Roku account, normally the ISO 3166-1 (2-letter) code.`

Reply:
Changed, thanks for the suggestion!